src/main/java/nopol_examples/nopol_example_4/NopolExample.java

```diff
@@ -29,4 +30,4 @@ public boolean canBeDividedby3(String a) {
 		boolean result = (Integer.parseInt(a)%3 == 0);
```

Comment:
This line (30) has been identified with a suspiciousness value of 12.70%.
The following failing tests covered this line: `nopol_examples.nopol_example_4.NopolExampleTest#test10`,`nopol_examples.nopol_example_4.NopolExampleTest#test5`